src/pages/loginOrRegister.js

```diff
@@ -0,0 +1,45 @@
+import { useState } from "react";
+import "./loginOrRegister.css";
+import { LoginForm } from "../components/loginForm";
+import { RegisterForm } from "../components/registerForm";
+
+function LoginOrRegister(props) {
+  const [activeForm, setActiveForm] = useState("login");
+  const [message, setMessage] = useState("");
+
+  let showLoginForm = (m = "") => {
+    setActiveForm("login");
+    if (typeof m === "string") setMessage(m);
+  };
```

Comment:
This function is doing two jobs ideally, one function should do only one job. After reading the name it's hard to perceive it receives a message argument too. You can directly pass `setMessage` as a prop to setMessage from child components.